logexporter/cmd/main.go

```diff
@@ -51,7 +51,8 @@ var (
 
 	// Node-type specific logfiles.
 	// Currently we only handle nodes, and neglect master.
-	nodeLogs = []string{"kube-proxy", "node-problem-detector", "fluentd"}
+	// NOTE(oxddr): cl2-* is prefix we expect from files created by clusterloader2
+	nodeLogs = []string{"kube-proxy", "node-problem-detector", "fluentd", "cl2-*"}
```

Comment:
Sorry for being so picking, I definitely don't want to block this, so I'm giving LGTM

But I think it would be better if we introduced a new flag for "extra-logfiles". By doing that we wouldn't have to build a  new version of logexporter every time we want to add new logfiles. 

/lgtm

Reply:
Done.